xbmc/Application.cpp

```diff
@@ -3478,12 +3478,16 @@ void CApplication::OnPlayBackSeekChapter(int iChapter)
 
 void CApplication::OnAVStarted(const CFileItem &file)
 {
+  CLog::LogF(LOGDEBUG, "CApplication::OnAVStarted");
+
   CGUIMessage msg(GUI_MSG_PLAYBACK_AVSTARTED, 0, 0);
   g_windowManager.SendThreadMessage(msg);
 }
 
 void CApplication::OnAVChange()
 {
+  CLog::LogF(LOGDEBUG, "CApplication::OnAVChange");
```

Comment:
Not sure about this as it might get spammy, well we could drop it down to the trace level?

Reply:
> well we could drop it down to the trace level?

good idea

Reply:
Sorry, as it turns out `DEBUG` is the lowest log level kodi has. Seems like I've been doing too much non core work :/

Reply:
nevermind. shove it in